javascriptv3/example_code/cognito/admin-create-user.js

```diff
@@ -0,0 +1,48 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3)
+
+Purpose:
+admin-create-user.js demonstrates how to create an user from Cognito SDK.
+
+*/
+

```

Comment:
You can remove lines 10 thru. 23 (these are deprecated internal references)

Reply:
Removed.